content/support-matrix/delfin/_index.md

```diff
@@ -10,17 +10,18 @@ tags: ["support matrix", "delfin", "drivers"]
+| IBM                                                 | Storewize, SVC            | 6.2–7.8                            | SSH                     | Block   | √                        | √            | √   | √          | √    | √    | NA    | NA    | NA          | NA    | √                    | √         | √           | √               | ×                   |                                                                                                                                                                                                         |   |
+| Hitachi                                             | VSP                       | 88-01-0X-XX/XX or later            | REST                    | Block   | √                        | √            | √   | √          | √    | √    | NA    | NA    | NA          | NA    | √                    | √         | √           | ×               | ×                   | 1\. The Configuration Manager REST service needs to be installed on the VSP devices.                                                                                                                    |   |
+| Huawei                                              | OceanStor                 | V3/V5                              | REST                    | Unified | √                        | √            | √   | √          | √    | √    | √     | √     | √           | √     | √                    | √         | √           | √               | √                   |                                                                                                                                                                                                         |   |
+| NetApp                                              | Cluster mode              | ONTAP 9.X                          | SSH                     | Unified | √                        | √            | √   | √          | √    | √    | √     | √     | √           | √     | √                    | √         | √           | √               | ×                   |                                                                                                                                                                                                         |   |
+| Simulated Sample Driver (for testing the framework) | NA                        | NA                                 | NA                      | NA      | √                        | √            | √   | √          | √    | √    | √     | √     | √           | √     | √                    | √         | √           | √               | √                   | 1\. Used only for framework testing purpose                                                                                                                                                             |   |                                                           |                                                          |
```

Comment:
At the end of page, we still have below statement
"Support matrix of drivers for Dell EMC VNX/VPLEX/Unity, IBM Storwize/SVC, Hitachi VSP, HPE 3PAR are currently not updated here."

Reply:
right , we have separate session for driver level updates, there we do not have these driver specific details of support I will currently remove it . we have only page for vmax and oceanstor